test/verify/machineslib.py

```diff
@@ -1136,6 +1163,92 @@ def installWithErrorTest(dialog):
                                          start_vm=True))
         # End of tests for import existing disk as installation option
 
+        # test PXE Source
+        if self.provider == "libvirt-dbus":
+            self.machine.execute("virsh net-destroy default && virsh net-undefine default")
+
+            if not ('debian' in self.machine.image or 'ubuntu' in self.machine.image):
+                # Disable selinux because it makes TFTP directory inaccesible and we don't want sophisticated configuration for tests
+                self.machine.execute("setenforce 0")
```

Comment:
If this works, it's ok. But it might be more robust to call `if type setenforce >/dev/null 2>&1; then setenforce 0; fi`.

Reply:
I replaced it with your script.